__init__.py

```diff
@@ -0,0 +1,14 @@
+# File: __init__.py
+#
+# Copyright (c) 2021-2022 Splunk Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under
+# the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+# either express or implied. See the License for the specific language governing permissions
+# and limitations under the License.
```

Comment:
License update is suggested by Static test failure.
reference: https://github.com/splunk-soar-connectors/maxmind/blob/next/maxmind_connector.py#L1